src/agent/src/device.rs

```diff
@@ -175,12 +175,12 @@ pub fn pcipath_to_sysfs(root_bus_sysfs: &str, pcipath: &pci::Path) -> Result<Str
             ));
```

Comment:
The first paragraph of your commit message is no terribly accurate.

Splitting the changes to baremount() into a preliminary patch would probably make this easier to review (and to rebase)

Reply:
@dgibson - Yep - currently doing that...